src/modules/app/containers/__tests__/__snapshots__/RouteContentSwitcher.spec.js.snap

```diff
@@ -0,0 +1,45 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`RouteContentSwitcher renderFailureLoadingComponents should render a FailureSwitcher if a payload contains an error 1`] = `
+<Translate(FailureSwitcher)
+  error={[Error: error]}
+/>
+`;
+
+exports[`RouteContentSwitcher renderFailureLoadingComponents should render a Spinner if data has not been fetched yet 1`] = `<ForwardRef />`;
```

Comment:
A Spinner and not a ForwardRef should be rendered

Reply:
Done